roles/commcare_analytics/templates/postgres/postgresql.conf.j2

```diff
@@ -115,7 +115,7 @@ ssl_key_file = '/etc/ssl/private/ssl-cert-snakeoil.key'
 
 # - Memory -
 
-shared_buffers = 128MB			# min 128kB
+shared_buffers = 2GB			# min 128kB
```

Comment:
It is recommended that postgres could get upto 25% of the machine memory.
On staging we are currently at 8GB and 32GB for production, so this change should be safe for both.